### ISSUE TITLE
post_processing example does not work

### ISSUE THREAD

**Issue by YoshieraHuang** [bevyengine/bevy#5033]
## Bevy version

caa16c5

## Relevant system information

Windows 11

```
AdapterInfo { name: "Radeon RX 590 Series", vendor: 4098, device: 26591, device_type: DiscreteGpu, backend: Vulkan }
```

## What you did

```
cargo run --example post_processing --release
```

## What went wrong

 Something should appear in the window, but the window is blank.

**Comment by alice-i-cecile** [comment]
Do the other shader examples work for you?

**Comment by mockersf** [comment]
duplicate of #5019, fixed by #5020
